### DISCUSSION TITLE
그래픽 함수 사용 가능한가요?

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#216] [5. Q&A]
Graphics 와 Shader 함수 사용 가능한가요? 
사용하는 다른 방법이 있나요?
그냥은 사용이 안되는데 이것만 사용되면 때깔곱게 셰이더팩을 만들 수 있을거같아요 :)

혹시 사용할 수 있는 방법이 전혀 없을까요? 리더보드때처럼 어떤 방식으로는 사용 가능하다든지... 히히

**Comment by hyeonjin-jo** [comment]
안녕하세요.

Shader는 기존 유니티 사용법과 동일하게 활용 가능합니다.

**Reply by x4cf** [reply]
네! 쉐이더에서는 가능하지만, 타입스크립트 내에서 Graphics.blit 함수를 사용하려고 하는데 단독으로 사용해서는 작동을 안하더라구요! 혹시 작동시킬 방법이 있을까요?

**Reply by x4cf** [reply]
유니티 생명주기의 OnRenderImage 함수도 사용가능한지 알고싶어요!

**Reply by crash-k** [reply]
안녕하세요.
말씀하신 Graphics.blit, OnRenderImage 포함 된 Graphics api 를 다음 버전에 제공 할 예정이고요.
다음버전은 제페토 앱 3.9.7 배포와 동시에 나갈 예정입니다.
참고 부탁드립니다.
감사합니다.

**Reply by Cha-SeongHun** [reply]
1.1.4 버전에선 정상동작했으나
1.1.7 버전에서는 OnRenderImage(src, dst) 함수 호출이 되지 않습니다.

의도적으로 제거된 사양인지, 이슈인지 확인해주시면 감사하겠습니다

**Reply by crash-k** [reply]
안녕하세요.
1.1.7 에 특별히 빠지진 않았습니다.
내부 테스트에서는 문제가 없는데요. 혹시 좀 더 오류 상황을 공유 해 주실 수 있을까요??

**Reply by x4cf** [reply]
인텔 맥 환경 제페토 월드 1.2.0 에서 함수는 정상적으로 호출됩니다 :D

**Reply by Cha-SeongHun** [reply]
Windows 에디터 환경 1.3.0 패키지에서 아래 스크립트 작성 및 배치 후 플레이 시 로깅이 되지 않습니다
```ts
import { Graphics, Material, RenderTexture } from "UnityEngine";
import { ZepetoScriptBehaviour } from "ZEPETO.Script";

export default class PostRenderer extends ZepetoScriptBehaviour {
    public material: Material;

    private OnRenderImage(src: RenderTexture, dest: RenderTexture) {
        console.log("[OnRenderImage]");
        Graphics.Blit(src, dest, this.material);
    }
}

```
